src/main/java/hive/player/entity/Player.java

```diff
@@ -0,0 +1,90 @@
+package hive.player.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import hive.player.exception.BlankIdException;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "tb_user_profile")
+public class Player {
+  @Id
+  @JsonIgnore
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  //is some DBMS like Oracle you should change the strategy
+  private Integer playerId;
+  @JsonIgnore
+  private String authenticatedUserId;
+
+  @JsonProperty
+  @Column(name = "login_alias", unique = true)
+  private String loginAlias;
+  @JsonProperty
+  @Column(name = "email")
+  private String email;
+  @JsonProperty
+  @Column(name = "telnumber")
+  private String telnumber;
+  @JsonProperty
+  @Column(name = "flavor_text")
+  private String flavorText;
+  @JsonProperty
+  @Column(name = "birthday")
+  private String birthday;
+
+  @Embedded
+  @JsonProperty
+  @Column(name = "options")
+  private PlayerOptions options;
+  @Embedded
+  @JsonProperty
+  @Column(name = "social")
+  private PlayerSocial social;
+
+  private Player() {
```

Comment:
```suggestion
  private Player() {
```
put next to the other constructors